bsp/imxrt1052-evk/drivers/drv_hwtimer.c

```diff
@@ -0,0 +1,195 @@
+/*
+ * File      : drv_hwtimer.c
+ * This file is part of RT-Thread RTOS
+ * COPYRIGHT (C) 2006-2013, RT-Thread Development Team
+ *
+ * The license and distribution terms for this file may be
+ * found in the file LICENSE in this distribution or at
+ * http://www.rt-thread.org/license/LICENSE
```

Comment:
For file header, please refer to:

https://github.com/RT-Thread/rtthread-specification/blob/master/bsp_skeleton/driver/board.h